obspy/core/tests/test_event.py

```diff
@@ -151,6 +154,24 @@ def test_plot_farfield_without_quiver_with_maps(self):
             ev.plot(kind=[['global'], ['ortho', 'beachball'],
                           ['p_sphere', 's_sphere']], outfile=ic.name)
 
+    def test_farfield_2xn_input(self):
+        """
+        Tests to compute P/S wave farfield radiation pattern using (theta,phi)
+        pairs as input
+        """
+        # Peru 2001/6/23 20:34:23:
+        mt = [2.245, -0.547, -1.698, 1.339, -3.728, 1.444]
+        theta = np.arange(0, 360, 60)
+        phi = np.zeros(len(theta))
+        rays = np.array([theta, phi])
+        result = farfield(mt, rays, 'P')
+        ref = np.array([[-0., 1.06567166, -2.26055006, 2.19679324, -0.44435406,
+                         -2.07785517], [-0., 0., -0., 0., -0., -0.],
+                        [-1.698, 3.32980367, -3.16993006, 1.64100194,
+                         -0.15311543,
+                        -0.04592479]])
+        self.assertTrue(np.allclose(result, ref))
```

Comment:
There is also an `np.testing.assert_allclose()` which does the same thing but provides nicer and more helpful error messages in case something goes wrong.